### PR TITLE
TRD-DigitTask   Simplified histogram view by merging ~ 180 plots onto 39 canvases

### PR DESCRIPTION
**Pull request by namalikgit**

### CODE REVIEW

File: Modules/TRD/src/DigitsTask.cxx [hunk 1]
```diff
@@ -211,102 +210,165 @@ void DigitsTask::fillLinesOnHistsPerLayer(int iLayer)
...
+  _cCluster->cd(4);
   mClsChargeTb.reset(new TH1F("Cluster/ClsChargeTb", "ClsChargeTb;", 30, -0.5, 29.5));
-  getObjectsManager()->startPublishing(mClsChargeTb.get());
-
-  mClsChargeTbCycle.reset(new TH1F("Cluster/ClsChargeTbCycle", "ClsChargeTb cleared per cycle;Cluster charge; Timebin", 30, -0.5, 29.5));
-  getObjectsManager()->startPublishing(mClsChargeTbCycle.get());
+  mClsChargeTb.get()->Draw();
 
+  _cCluster->cd(5);
   mClsAmp.reset(new TH1F("Cluster/ClsAmp", "Amplitude of clusters;Amplitdu(ADC);Counts", 200, -0.5, 1999.5));
```
Comment (bazinski) [inline Modules/TRD/src/DigitsTask.cxx:259]: Amplitude(ADC)

**Comment by bazinski** [comment]
line 762 : 
mClusterAmplitudeChamber->Fill(sum, chamber);
should go after the subsequent curly bracket.

**Comment by Barthelemy** [comment]
@bazinski could you approve it ?

**Review by bazinski — APPROVED** [review]

File: Modules/TRD/src/DigitsTask.cxx [hunk 2]
```diff
@@ -211,102 +210,165 @@ void DigitsTask::fillLinesOnHistsPerLayer(int iLayer)
...
+  _cCluster_1 = cCluster;
+
+  _cCluster_1->cd(1);
   mNCls.reset(new TH1F("Cluster/NCls", "Total number of clusters per sector", 18, -0.5, 17.5));
   mNCls->SetTitle("Total number of clusters per sector;Sector;Counts");
-  getObjectsManager()->startPublishing(mNCls.get());
+  mNCls.get()->Draw();
 
+  _cCluster_1->cd(2);
   mNClsAmp.reset(new TH1F("Cluster/NClsAmp", "Total number of clusters per sector;Sector;Counts", 200, -0.5, 199.5));
```
[the remark this answers is not in the document]
Comment (namalikgit) [reply Modules/TRD/src/DigitsTask.cxx]: @bazinski yes I removed  mNClsAmp and added a cluster per layer histo, look at  the changes  if that is correct

**Review by Barthelemy — APPROVED** [review]

**Comment by martenole** [comment]
Hi @Barthelemy @bazinski and @shahor02 (fyi)
this change introduced quite a lot of warnings both at the synchronous stage (see FLP IL) and at the async stage (see https://alimonitor.cern.ch/users/download.jsp?view=true&path=%2Falice%2Fcern.ch%2Fuser%2Fa%2Falidaq%2Frecycle%2Falien-job-2851679713%2Fstdout) similar to
```
[WARN] Object 'PulseHeight/PulseHeightPerSuperModule_SM-12-17' with type 'TCanvas' is not one of the mergeable types, skipping
```
Shall we revert this change?

**Comment by bazinski** [comment]
@martenole yes that would be my suggestion, if it can simply be done here, there cool even easier.

**Comment by martenole** [comment]
Ok, PR is here: https://github.com/AliceO2Group/QualityControl/pull/1789

**Pull request merged**